.github/workflows/lint-cron.yml

```diff
@@ -11,15 +11,15 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install Node v16
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: 16
 
       - name: Install dependencies
-        run: npm ci
+        run: yarn install --frozen-lockfile
```

Comment:
Intentional to have no caching of `node_modules`? Otherwise you can change this and similar to:
```yaml
      - name: Restore CI Cache
        uses: actions/cache@v2.1.6
        id: cache-restore
        with:
          path: node_modules
          key: ${{ runner.os }}-16-${{ hashFiles('**/yarn.lock') }}

      - name: Install Dependencies if Cache Miss
        if: ${{ !steps.cache-restore.outputs.cache-hit }}
        run: yarn --frozen-lockfile
```

Reply:
Intended, I hate dealing with CI caching and take the longer install time.

GitLab gave me nightmares with that, never again. I have trust issues.